Sprint-1/interpret/to-pounds.js

```diff
@@ -24,4 +29,16 @@ console.log(`£${pounds}.${pence}`);
 // Try and describe the purpose / rationale behind each step
 
 // To begin, we can start with
-// 1. const penceString = "399p": initialises a string variable with the value "399p"
+// 1. const penceString = "399p": initializes a string variable with the value "399p"
+// 3. initializes a variable that removes the "p" at the end, it uses method .substring
+// which returns the part of the string corresponding to the index and end index
+//-1 excluding the p
+//9. uses the method padstart to add "0" to the start of string penceStringWithoutTrailingP,
```

Comment:
After line 9, what would `paddedPenceNumberString` be if `penceString` was "1234p" initially?
After line 9, what would `paddedPenceNumberString` be if `penceString` was "4p" initially?

Reply:
It will return 1234
It will return 004